src/test/java/Homework17.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+@Test
+    public void addSongToPlaylist() throws InterruptedException{
+
+String newSongAddedNotification = "Added 1 song into";
+
```

Comment:
I suggest you need to format the code to make the code look nicer and easier to see.
You can use shortcut to format code. Refer in this link (https://www.jetbrains.com/idea/guide/tips/reformat-file/#:~:text=You%20can%20use%20%E2%8C%98%E2%8C%A5,L%20(Windows%2FLinux).)